css/style.css

```
@@ -154,7 +155,7 @@ body {
 
 p {
     font-family: 'Roboto Slab', serif;
-    text-indent: 15pt;
+    /* text-indent: 15pt; */
```

Eu gosto do recuo de primeira linha 🥺 
```suggestion
    text-indent: 15pt;
```